backend/src/models/Model.ts

```diff
@@ -40,6 +40,7 @@ export interface ModelCardInterface {
 
 export interface Settings {
   ungovernedAccess: boolean
+  allowTemplating: boolean
   mirror: { sourceModelId?: string; destinationModelId?: string }
```

Comment:
Not your doing, but this property isn't defined in the frontend `EntryInterface`. Should it be?

Reply:
It's declared in the next ticket. My bad

Reply:
To be clear, I'm talking about the `mirror` property here. It looks like it was added to the backend interface at some point in the past, but wasn't replicated in the frontend.

You've correctly added your new `allowTemplating` property to the frontend interface in both of your PRs, so no need to worry about that.

Reply:
We haven't done any work on the mirrored model stuff in the frontend yet, and I'm assuming updating the frontend interface can wait until then?